docs/reference/indices/create-index.asciidoc

```diff
@@ -177,28 +175,3 @@ PUT test?wait_for_active_shards=2
 
 A detailed explanation of `wait_for_active_shards` and its possible values can be found
 <<index-wait-for-active-shards,here>>.
-
-[float]
-=== Skipping types
```

Comment:
Should we have the reverse section as well explaining how to keep types although deprecated in case you still need them?

Reply:
I'm inclined to keep the main REST documentation free from references to types, as we have for the other APIs. I have a standing TODO to update `removal-of-mapping-types.asciidoc` with the new plan, including some examples of using`include_type_name`. Perhaps I could link to that page from here, as we do for the index API documentation?